Sources/IfTag.swift

```diff
@@ -118,7 +118,12 @@ final class IfExpressionParser {
   private init(components: ArraySlice<String>, tokenParser: TokenParser, token: Token) throws {
     var parsedComponents = Set<Int>()
     var bracketsBalance = 0
-    self.tokens = try zip(components.indices, components).compactMap { (index, component) in
+    #if swift(>=4.1)
```

Comment:
maybe it would be better to add an extension that defines `compactMap` on collections for swift 4.1 rather than checking it in place? will be also easier to remove later.